mono/utils/mono-threads-api.h

```diff
@@ -72,13 +72,13 @@ mono_threads_assert_gc_unsafe_region (void);
 MONO_API gpointer
 mono_threads_enter_gc_safe_region (gpointer *stackdata);
 
-gpointer
+MONO_API gpointer // used by profiler
```

Comment:
`MONO_PROFILER_API` is what you want.

Reply:
I tried that first and it did not compile. I can try again.